src/modules/languages/react/LanguageSettingsPage.tsx

```diff
@@ -223,7 +239,7 @@ export default async function LanguageSettingsPage({
 
 async function fetchTranslations(
   languageCode: string,
-): Promise<{ id: string; name: string }[]> {
+): Promise<{ id: string; english_name: string }[]> {
```

Comment:
This function didn't need to change. The translations are in a separate system from the list of languages we are tracking